.gitattributes

```diff
@@ -0,0 +1,17 @@
+# https://php.watch/articles/composer-gitattributes
+
+# Exclude build/test files from archive
+/tests            export-ignore
+/.gitattributes   export-ignore
+/.gitignore       export-ignore
+/.scrutinizer.yml export-ignore
+/.travis.yml      export-ignore
+/codecov.yml      export-ignore
+/phpcs.xml        export-ignore
+/phpmd.xml        export-ignore
+/phpstan.neon     export-ignore
+/phpunit.xml.dist export-ignore
```

Comment:
I want for my project:
```
/.gitignore       
/.scrutinizer.yml 
/.travis.yml      
/codecov.yml      
/phpcs.xml        
/phpmd.xml        
/phpstan.neon     
/phpunit.xml.dist 
```

Reply:
Fixed with #63. @kenjis Thank you for the taking time.